scripts/results/session-segments.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+#.....................................................................
+# Copyright (c) 2016-2020, Regents of the University of Arizona.
+# Author: Wenkai Zheng <wenkaizheng@email.arizona.edu>
+#         Chavoosh Ghasemi <chghasemi@cs.arizona.edu>
+
+# You should have received a copy of the GNU General Public License along with
+# this script e.g., in COPYING.md file. If not, see <http://www.gnu.org/licenses/>.
+#
+#
+# DESCRIPTION:
+# Feed this file with the collected log file of the consumers,
+# on the server side. You can find this file on the machine
+# that runs the stats-collector tool.
+#
+# The output is a plot, showing the number received Data
+# packets by the consumer during *each*
+# video playing back (i.e, each session).
```

Comment:
Fix the line-wrapping. One new line is enough.

Reply:
Did you mean just two lines?  I changed it to
#The output is a plot, showing the number received Data
# packets by the consumer during *each* video playing back (i.e, each session).

Reply:
Yes. The lines are too short now.